### ISSUE TITLE
Demo app - opening the push menu for already opened side menu

### ISSUE THREAD

**Issue by girdhariag** [rajasegar/ember-slide-push-menu#17]
When we try to open the push menu while the side menu is already open, the demo behaves weirdly.

![push-menu](https://user-images.githubusercontent.com/25173551/92847556-e65c2100-f406-11ea-9690-88a8849a9591.gif)

**Comment by rajasegar** [comment]
Yes @girdhariag seems like we need to do some cleanup before opening any menu.
